apps/example/messages/en.json

```diff
@@ -0,0 +1,20 @@
+{
```

Comment:
It's easier to maintain flat translation file like

`admin.actions.user.email.title`

Reply:
This nested notation is the only one supported by next-intl